src/shared/interfaces/json-schema.ts

```diff
@@ -0,0 +1,36 @@
+import { SchemaOption } from './schema-option';
+
+export interface JSONSchema extends SchemaOption {
```

Comment:
Where did you get the list from? In case you missed some key, there is this example https://gist.github.com/enriched/c84a2a99f886654149908091a3183e15

Do we need to add $ref ?